tests/unit/src/test/scala/tests/codeactions/StringActionsLspSuite.scala

```diff
@@ -0,0 +1,77 @@
+package tests.codeactions
+
+import scala.meta.internal.metals.codeactions.StringActions
+
+class MultlineStringLspSuite extends BaseCodeActionLspSuite("multilineString") {
+
+  check(
+    "empty-string",
+    """|package a
+       |
+       |object A {
+       |  val str = "<<>>"
+       |}
+       |""".stripMargin,
+    s"${StringActions.title}",
+    """|package a
+       |
+       |object A {
+       |  val str = '''|'''.stripMargin
```

Comment:
```suggestion
       |  val str = $tripleQuote|$tripleQuote.stripMargin
```

Reply:
I find it less readable actually, do you think it is really important ?

Reply:
Yeah, you're right. It's more readable this way. Let's leave it.